host/3.0/buster/amd64/java/java11/java11-slim.Dockerfile

```diff
@@ -19,6 +20,10 @@ RUN EXTENSION_BUNDLE_VERSION=1.8.1 && \
     EXTENSION_BUNDLE_FILENAME=Microsoft.Azure.Functions.ExtensionBundle.1.8.1_linux-x64.zip && \
     apt-get update && \
     apt-get install -y gnupg wget unzip && \
+    wget https://aka.ms/download-jdk/microsoft-jdk-${JAVA_VERSION}-linux-x64.tar.gz && \
+    mkdir -p /usr/lib/jvm && \
+    tar xvzf microsoft-jdk-${JAVA_VERSION}-linux-x64.tar.gz -C /usr/lib/jvm &&\
```

Comment:
For consistency sake, please consider replacing tar with unzip. Just because the other commands here use that pattern

Reply:
For linux jdk the jdk team only provide the tar binaries. 
msft-openjdk-11: https://docs.microsoft.com/en-us/java/openjdk/older-releases#openjdk-11
adoptium-jdk-8: https://github.com/adoptium/temurin8-binaries/releases